muselsl/cli.py

```diff
@@ -0,0 +1,157 @@
+#!/usr/bin/python
+import sys
+import argparse
+
+
+class main:
+    def __init__(self):
+        parser = argparse.ArgumentParser(
+            description='Python package for streaming, recording, and visualizing EEG data from the Muse 2016 headset.',
+            usage='''muselsl <command> [<args>]
+    Available commands:
+    list        List available Muse devices.
+                -b --backend    BLE backend to use. can be auto, bluemuse, gatt or bgapi.
+                -i --interface  The interfact to use, 'hci0' for gatt or a com port for bgapi.
+
+    stream      Start an LSL stream from Muse headset.
+                -a --address    Device MAC address.
+                -n --name       Device name (e.g. Muse-41D2).
+                -b --backend    BLE backend to use. can be auto, bluemuse, gatt or bgapi.
+                -i --interface  The interfact to use, 'hci0' for gatt or a com port for bgapi.
+
+    view     Visualize EEG data from an LSL stream.
+                -w --window     Window length to display in seconds.
+                -s --scale      Scale in uV.
+                -r --refresh    Refresh rate in seconds.
+                -f --figure     Window size.
+                -v --version    Viewer version (1 or 2) - 1 is the default stable version, 2 is in development (and takes no arguments).
+
+    record   Record EEG data from an LSL stream.
+                -d --duration   Duration of the recording in seconds.
+                -f --filename   Name of the recording file.
+                -dj --dejitter  Whether to apply dejitter correction to timestamps.
+
+    record_direct      Record data directly from Muse headset (no LSL).
+                -a --address    Device MAC address.
+                -n --name       Device name (e.g. Muse-41D2).
+                -b --backend    BLE backend to use. can be auto, bluemuse, gatt or bgapi.
+                -i --interface  The interfact to use, 'hci0' for gatt or a com port for bgapi.
+        ''')
```

Comment:
Could the `duration` argument also be added to the `record_direct` option?

Reply:
Good catch. I added this feature